source/common/quic/envoy_quic_client_stream.cc

```diff
@@ -44,16 +44,6 @@ EnvoyQuicClientStream::EnvoyQuicClientStream(
          "Send buffer limit should be larger than 8KB.");
 }
 
-EnvoyQuicClientStream::EnvoyQuicClientStream(
-    quic::PendingStream* pending, quic::QuicSpdyClientSession* client_session,
-    quic::StreamType type, Http::Http3::CodecStats& stats,
-    const envoy::config::core::v3::Http3ProtocolOptions& http3_options)
-    : quic::QuicSpdyClientStream(pending, client_session, type),
-      EnvoyQuicStream(
-          static_cast<uint32_t>(GetReceiveWindow().value()), *filterManagerConnection(),
-          [this]() { runLowWatermarkCallbacks(); }, [this]() { runHighWatermarkCallbacks(); },
-          stats, http3_options) {}
-
```

Comment:
These changes seem unrelated.  Was this code unused?

Reply:
These are only called by server push stream.